test/dynamo/test_optimizers.py

```diff
@@ -48,7 +48,9 @@ class OptimizerTests(torch._dynamo.test_case.TestCase):
     # furthermore, the break is inside a for loop, so we bail on the frame
     # entirely.  This is basically an xfail; if the frame count goes up
     # you done good
-    test_radam = make_test(torch.optim.RAdam, exp_graph_count=0)
+    test_radam = torch._dynamo.testing.skip_if_pytest(
```

Comment:
Curios why? There are some tests which are generated below which may need this as well

Reply:
I think pytest rewrites python asserts at the bytecode level which triggers an extra graph break.  Didn't look into it in detail.